scripts/version

```diff
@@ -4,13 +4,17 @@ if [ -n "$(git status --porcelain --untracked-files=no)" ]; then
     DIRTY="-dirty"
 fi
 
-COMMIT=$(git rev-parse --short HEAD)
 GIT_TAG=${DRONE_TAG:-$(git tag -l --contains HEAD | head -n 1)}
 
 if [[ -z "$DIRTY" && -n "$GIT_TAG" ]]; then
     VERSION=$GIT_TAG
 else
-    VERSION="${COMMIT}${DIRTY}"
+    # Make sure tags are available in the CI environment
+    readarray -t TAGS < <(git tag -l)
+    if [[ ${#TAGS[@]} == 0 ]]; then
+        git fetch -t
+    fi
+    VERSION="$(git describe --tags)"
```

Comment:
It is not possible to build a "dirty" gitjob version because of https://github.com/rancher/gitjob/blob/master/scripts/validate-ci.
But I will add the same logic to be consistent.

Reply:
@thardeck the `+dirty` setup will work if people run this outside of CI. I think that's an intent.